corehq/apps/widget/util.py

```diff
@@ -0,0 +1,11 @@
+from corehq.apps.widget.models import DialerSettings
+
+def domain_uses_dialer(domain):
+    try:
```

Comment:
@orangejenny I'd like to make it possible for "enterprise spaces to check a box that says "Use Enterprise Dialer settings" which this method would then use to look up the chain at the "control" space to get its settings and mirror them back down. Is that consistent with how enterprise managed domains work?

Reply:
Yes and no. You can look up the controlling domain's name with `DomainPermissionsMirror.source_domain(domain)` and then pull its settings, yes. But enterprise domains don't have any additional config. So your options, from low to high effort, are:
1) Don't add a config at all
2) Add a boolean field `domain_uses_dialer` to `DomainPermissionsMirror` and make it visible in django admin
3) Add the boolean field and also add a checkbox to the `DomainPermissionsMirrorView`, which is currently a read-only UI.

I think any of these is fine. `DomainPermissionsMirror` is new enough and invisible enough that no one is using it at this point except the projects it was built for.